src/NServiceBus.Core/Hosting/Helpers/AssemblyScannerConfiguration.cs

```diff
@@ -24,7 +24,7 @@ public class AssemblyScannerConfiguration
         /// <summary>
         /// Defines whether nested directories should be included in the assembly scanning process. Default value is <code>false</code>.
         /// </summary>
-        public bool ScanAssembliesInNestedDirectories { get; set; } = false;
+        public bool ScanAssembliesInNestedDirectories { get; set; }
```

Comment:
> but as a user if I'm interested in the default value, I'd immediately see the default value when looking at the source and probably trust this value to remain unless something is configured explicitly

so you are talking about a person who is advanced enough to get and review our source but not advanced enough to know what the default field inits are.

My change is consistent with the majority of the code base. we do not init every other field to null "just in case a someone who doesnt knoe the difference reviews the code"